error.go

```diff
@@ -2,38 +2,42 @@
 // and efficient error handling.
 package microerror
 
-import (
-	"fmt"
-
-	"github.com/juju/errgo"
-)
-
 var (
-	// MaskAny is a simple error masker. Masked errors act as tracers within the
-	// source code. Inspecting an masked error shows where the error was passed
-	// through within the code base. This is gold for debugging and bug huntin.
-	MaskAny = errgo.MaskFunc(errgo.Any)
+	Default = NewErrgoHandler(ErrgoHandlerConfig{
+		CallDepth: 1,
```

Comment:
> Then the default must be private. 

The whole point of it is to have that public var. That way if a user uses for example `operatorkit` she can change the implementation even though `operatorkit` uses `microkit.Mask` global functions.

The `Default` name can be misleading I agree. 

I see 3 solutions here:

1. Changing the name of `Default` to `InternalHandler` + godoc.
2. Creating unexported `mask` and `maskf` methods taking `callDepth` and removing `CallDepth` from `ErrgoHandlerConfig`.
3. Adding `MaskCallDepth(err error, callDepth int)` and `MaskfCallDepth(err error, f string, v ...interface{})` and removing `CallDepth` from `ErrgoHandlerConfig`.

I like 1. most, but I'm fine with any.